sdk/eventgrid/Azure.Messaging.EventGrid/api/Azure.Messaging.EventGrid.netstandard2.0.cs

```diff
@@ -153,31 +153,31 @@ public partial class ACSChatThreadWithUserDeletedEventData : Azure.Messaging.Eve
         public string DeletedBy { get { throw null; } }
         public System.DateTimeOffset? DeleteTime { get { throw null; } }
     }
-    public partial class AcssmsDeliveryAttemptProperties
+    public partial class AcsSmsDeliveryAttemptProperties
```

Comment:
Do we want to call out in the CHANGELOG.md that the casing of this type has changed between beta2 and beta3?

Reply:
Was planning on doing a separate PR for changelog that will include all of the changes for the upcoming beta - CloudEvent attribute stuff, Topic parsing fix for EGEvents, rename DataSerializer to Serializer.